### ISSUE TITLE
Use Cucumber Expressions?

### ISSUE THREAD

**Issue by aslakhellesoy** [cbliard/coffee-machine-javascript#1]
Cucumber Open no longer encourages the use of regular expressions. All snippets are Cucumber Expressions.

In order to be consistent and avoid confusing users I think we should avoid using Regular Expressions for our example projects and use Cucumber Expressions instead :-)

**Comment by cbliard** [comment]
You are totally right

Le jeu. 28 janv. 2021 à 11:10, Aslak Hellesøy <notifications@github.com> a
écrit :

**Comment by cbliard** [comment]
And that's not the only issue. We should also stay away from the Connextra template (As a... I want to... So that...). Seb and Steve told me that as a feature is not a user story, it does not really make any sense.

Our whole CucumberStudio documentation and hiptest-publisher templates have to be updated to fix this.

For this repository, I used regular expressions because that's what hiptest-publisher is exporting at the moment. I had to get something quick and running for the demo. Thanksfully, the community [has proposed enhancements](https://github.com/hiptest/hiptest-publisher/pull/205) and the TypeScript, Java, and Groovy exports will use Cucumber Expressions in next release of hiptest-publisher.

For next demo I'll make sure to update it, as well as the one on [cucumber/coffee-machine-javascript](https://github.com/cucumber/coffee-machine-javascript).

**Comment by aslakhellesoy** [comment]
Agreed on removing the connextra template. That said, I think the description ought to describe who the feature is for, and why it exists. I prefer simple prose without a particular template - whatever makes sense.
